packages/store/addon/-private/identifiers/cache.ts

```diff
@@ -157,18 +163,31 @@ export class IdentifierCache {
       return resource;
     }
 
+    let lid = coerceId(resource.lid);
+    let identifier: StableRecordIdentifier | undefined = lid !== null ? this._cache.lids[lid] : undefined;
```

Comment:
Can we simplify this line please, it's pretty hard to read for me at least

Reply:
If identifier is stableRecordIdentifier, wouldn't it have been returned on line 163?

Reply:
Resolving because this is outdated.